xlive/H2MOD.cpp

```diff
@@ -973,9 +973,10 @@ bool __cdecl OnPlayerSpawn(int a1)
 
 	int PlayerIndex = a1 & 0x000FFFF; 
 
-	if (b_Infection)
+	if (b_Infection) {
```

Comment:
the way i read the old code, we called spawnPlayer regardless of this if block, are there any edge cases here where spawn player does not get called?

Reply:
no, i changed because the second SpawnPlayer func from inf class was getting called even though infection wasn't enabled